js/all-the-ways-to-define-functions-in-js/all-the-ways-to-define-functions-in-js.md

```diff
@@ -0,0 +1,39 @@
+# All the ways to define functions in JavaScript
+
+## Introduction
+
+Function is a set of statements or block of code to perform a specific task and return a value.They are reusuable and make your program modular and efficient.
```

Comment:
The function is a set of statements or blocks of code to perform a specific task and return a value. They are reusable and make your program modular and efficient.